tests/syft/lib/tenseal/tenseal_bfvvector_test.py

```diff
@@ -0,0 +1,217 @@
+# third party
+import pytest
+
+# syft absolute
+import syft as sy
+
+from typing import Any
+
+
+@pytest.fixture(scope="function")
+def context() -> Any:
+    import tenseal as ts
```

Comment:
Q: Why don't we import this at the top of the file? (and also the ```sy.load_lib``` there?)

Reply:
Thinking more into this. Can't we do something with ```pytest.importorskip("tenseal")```

Reply:
Good idea, this seems to work for me:
```
# stdlib
from typing import Any

# third party
import pytest

# syft absolute
import syft as sy

ts = pytest.importorskip("tenseal")
sy.load_lib("tenseal")
```

Then as long as all the tests correctly specify their minimum requirements the pytest import will skip and the load_lib will skip.